src/main/java/frc/robot/led/LEDManager.java

```diff
@@ -0,0 +1,66 @@
+package frc.robot.led;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import frc.robot.RobotConstants;
+
+import java.util.ArrayList;
+
+public class LEDManager {
+    private static LEDManager instance = null;
+    private final ArrayList<Integer> offsets = new ArrayList<>();
+    private int length = 0;
+    private AddressableLEDBuffer ledBuffer;
+    private AddressableLED ledStrip;
+
+    public static LEDManager getInstance() {
```

Comment:
this should by synchronized
```suggestion
    public static synchronized LEDManager getInstance() {
```